specification/web/resource-manager/Microsoft.DomainRegistration/stable/2024-11-01/Domains.json

```diff
@@ -0,0 +1,1616 @@
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DomainRegistration/domains/{domainName}/domainOwnershipIdentifiers/{name}": {
+      "get": {
+        "tags": [
+          "Domains"
+        ],
+        "summary": "Get ownership identifier for domain",
+        "description": "Description for Get ownership identifier for domain",
+        "operationId": "Domains_GetOwnershipIdentifier",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "domainName",
+            "in": "path",
+            "description": "Name of domain.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of identifier.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DomainOwnershipIdentifier"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Domain Ownership Identifier": {
+            "$ref": "./examples/GetDomainOwnershipIdentifier.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Domains"
+        ],
+        "summary": "Creates an ownership identifier for a domain or updates identifier details for an existing identifier",
+        "description": "Description for Creates an ownership identifier for a domain or updates identifier details for an existing identifier",
+        "operationId": "Domains_CreateOrUpdateOwnershipIdentifier",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "domainName",
+            "in": "path",
+            "description": "Name of domain.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of identifier.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "domainOwnershipIdentifier",
+            "in": "body",
+            "description": "A JSON representation of the domain ownership properties.",
```

Comment:
>A JSON representation of the domain ownership properties. [](http://example.com/codeflow?start=28&length=57)

No need to mention JSON here, e.g. "Domain ownership identifier object to create or update."